app/src/main/java/fr/neamar/kiss/db/DB.java

```diff
@@ -8,19 +8,21 @@
 class DB extends SQLiteOpenHelper {
 
     private final static String DB_NAME = "kiss.s3db";
-    private final static int DB_VERSION = 6;
+    private final static int DB_VERSION = 7;
 
     DB(Context context) {
         super(context, DB_NAME, null, DB_VERSION);
     }
 
     @Override
     public void onCreate(SQLiteDatabase database) {
-        database.execSQL("CREATE TABLE history ( _id INTEGER PRIMARY KEY AUTOINCREMENT, query TEXT, record TEXT NOT NULL)");
+        // `query` is a keyword so we escape it. See: https://www.sqlite.org/lang_keywords.html
+        database.execSQL("CREATE TABLE history ( _id INTEGER PRIMARY KEY AUTOINCREMENT, \"query\" TEXT, record TEXT NOT NULL)");
```

Comment:
Why did this work before?

Reply:
It still works even if it's wrong. It may not work in the future tho'